vlanBridge.go

```diff
@@ -219,6 +224,24 @@ func (vl *VlanBridge) sendGARPAll() {
 	}
 }
 
+// Update global config
+func (vl *VlanBridge) GlobalConfigUpdate(cfg OfnetGlobalConfig) error {
+	if vl.agent.arpMode == cfg.ArpMode {
+		log.Errorf("no change in ARP mode %s", vl.agent.arpMode)
+	} else {
+		vl.agent.arpMode = cfg.ArpMode
+		add := (vl.agent.arpMode == "proxy")
+		vl.updateArpRedirectFlow(add)
+
+		if cfg.ArpMode == "flood" {
```

Comment:
Not changing the function name as it is used during the init as well. But moved the trigger GARP into the update function. @DivyaVavili PTAL